### PR TITLE
ci: Update auto-assign-issues.yml

### PR DESCRIPTION
**Pull request by Zidious**
Update to current board (automatically gets added to backlog)

### CODE REVIEW

File: .github/workflows/auto-assign-issues.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@ jobs:
       - name: Auto assign axe-core integration issues to Axe API team project board
         uses: srggrs/assign-one-project-github-action@1.2.1
         with:
-          project: https://github.com/orgs/dequelabs/projects/53
+          project: https://github.com/orgs/dequelabs/projects/66
           column_name: 'Ungroomed'
```
Comment (stephenmathieson) [inline .github/workflows/auto-assign-issues.yml:18]: Does this column exist? 🤔

**Review by stephenmathieson — APPROVED** [review]
LGTM assuming the "Ungroomed" column exists.

**Comment by stephenmathieson** [comment]
Since the column was renamed, do we need to go update every other repository that uses this?

Also, do we actually need this?

**Comment by Zidious** [comment]
> Since the column was renamed, do we need to go update every other repository that uses this?
> 
> Also, do we actually need this?

This can be removed

**Pull request closed without merging**